content/en/docs/reference/glossary/jwt.md

```diff
@@ -0,0 +1,19 @@
+---
+title: JSON Web Token (JWT)
+id: jwt
+date: 2022-12-06
+full_link: https://www.rfc-editor.org/rfc/rfc7519
+short_description: >
+  A digitally signed JSON object that can be used for authentication.
+
+aka:
+tags:
+- security
+- architecture
+---
+ A JSON object that can be digitally signed and used for authentication.
```

Comment:
BTW, JWTs are _not_ JSON objects. Usually they contain three separate JSON objects in a [custom format](https://www.rfc-editor.org/rfc/rfc7516#page-8).

Reply:
Applied your suggestion. is the "digitally signed" thing needed?

Reply:
I'd include it. JWT signing is an important feature.